pipeline/README.md

```diff
@@ -137,56 +73,51 @@ individual components; scripts to process plain text files from the
 command line; and .jar files for the libraries used by the pipeline and
 its components.
 
-The Illinois NLP Pipeline package sets default configuration options for
-all its components.  If you want to experiment with different settings,
-we recommend checking out the project from [github](https://github.com/IllinoisCogComp/illinois-cogcomp-nlp) -- see the section on Programmatic Use.
-
-## Dependencies
-If this package is used in maven, please add the following dependencies with proper repositories.
-```xml
-<dependencies>
-    <dependency>
-        <groupId>edu.illinois.cs.cogcomp</groupId>
-        <artifactId>illinois-nlp-pipeline</artifactId>
-        <version>#VERSION</version>
-    </dependency>
-</dependencies>
-<repositories>
-    <repository>
-        <id>CogcompSoftware</id>
-        <name>CogcompSoftware</name>
-        <url>http://cogcomp.cs.illinois.edu/m2repo/</url>
-    </repository>
-</repositories>
```

Comment:
Moved it to where start talking about "programmatic usage", after the "commandline usage"